internal/testutils/test.go

```diff
@@ -32,3 +33,8 @@ func FailTestOnAnyError(t *testing.T, errors []error, errorMessage string) {
 	}
 	t.FailNow()
 }
+
+// Must verifies that the error passed is not nil or stops execution
+func Must(t *testing.T, err error) {
```

Comment:
why not use `require.NoError` directly?
this wrapper seems a bit unnecessary.

Reply:
🤦